engine/src/gfx/cockpit_gfx.cpp

```diff
@@ -846,8 +849,9 @@ void DrawTurretTargetBoxes(const Radar::Sensor& sensor)
         Radar::Track track = sensor.CreateTrack(target, Loc);
         if (configuration()->graphics.hud.draw_nav_symbol) {
             GFXColor4f( 1, 1, 1, 1 );
-            DrawNavigationSymbol( subunit->GetComputerData().NavPoint, CamP, CamQ,
-                                 CamR.Cast().Dot( (subunit->GetComputerData().NavPoint).Cast()
+            Vector nav_point = subunit->GetNavPoint();
```

Comment:
Vector is not available in lower libraries. It is currently in lib_gfx.
CoreVector is available in lib_resource. We use it and a convenience function GetNavPoint to convert.

Reply:
CoreVector? Where's that?

Reply:
It's a very simple implementation of a struct with i,j,k.
I used it originally to refrain from including Vector in lib_damage, which would necessitate pulling all of gfx with it.
It's in lib_resource, which should really be renamed lib_core, as it houses some core stuff.